control-protocol.md

```diff
@@ -373,17 +379,48 @@ program start for example.
   # way as the info-request id.
   "id" : "hostname=uuid",
 
-  # the RePlied sequence number from the sender
-  "seq-rp" : <uint64_t>
+  # the RePlied sequence number from the sender, the number is encoded as as
+  # string. (E.g. "seq-rp" : "392192")
+  "seq-rp" : "<uint64_t>"
 
   # list of supported modules, the entries must point to empty dictionaries
   # for now. Later the empty dictionaries can be filled if additional
-  # information is required.
+  # information is required. The module specific info within the dictionary is
+  # specified in the particular module specification. E.g. the "tcp-goodput"
+  # module has a section which specifies allowed values for the tcp-goodput
+  # module.
   "modules" : {
      "udp-goodput" : { },
      "tcp-goodput" : { },
   }
 
+  # The info-reply SHOULD return a "control-protocol" block where all supported
```

Comment:
using "control-protocol" we can switch from Multicast to unicast after discovery phase right?